qiita_pet/test/test_util.py

```diff
@@ -28,6 +28,14 @@ def test_generate_param_str(self):
                "<b>sortmerna_coverage:</b> 0.97")
         self.assertEqual(obs, exp)
 
+    def test_is_local_connection(self):
+        self.assertTrue(is_local_connection('127.0.0.1'))
+        self.assertTrue(is_local_connection('localhost'))
+        self.assertTrue(is_local_connection('127.0.0.1:21174'))
+
+        self.assertFalse(is_local_connection('10.0.0.1'))
```

Comment:
It'd be more correct then if the function name was `is_localhost` as that
has a specific meaning. Should also verify that `::1` is resolved as
expected.

On Mon, Aug 10, 2015 at 10:05 AM, Antonio Gonzalez <notifications@github.com
> wrote:

> In qiita_pet/test/test_util.py
> <https://github.com/biocore/qiita/pull/1412#discussion_r36648859>:
>
> > @@ -28,6 +28,14 @@ def test_generate_param_str(self):
> >                 "<b>sortmerna_coverage:</b> 0.97")
> >          self.assertEqual(obs, exp)
> >
> > +    def test_is_local_connection(self):
> > +        self.assertTrue(is_local_connection('127.0.0.1'))
> > +        self.assertTrue(is_local_connection('localhost'))
> > +        self.assertTrue(is_local_connection('127.0.0.1:21174'))
> > +
> > +        self.assertFalse(is_local_connection('10.0.0.1'))
>
> That's a good point but is also the assumption made since the beginning.
> Note that this is a "nice" behavior as we can test as localhost and as
> external connection from the same machine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1412/files#r36648859>.
>